example.js

```diff
@@ -0,0 +1,353 @@
+var mutate = require('xtend/mutable')
+var expose = require('choo-expose')
+var css = require('sheetify')
+var html = require('bel')
+var choo = require('./')
+
+css('todomvc-common/base.css')
+css('todomvc-app-css/index.css')
+
+if (module.parent) {
+  exports.todoStore = todoStore
```

Comment:
Might this be overkill for an example app? Or is there some reason to include this?